.github/scripts/fix-sonar-shell-parse.py

```
@@ -0,0 +1,16 @@
+text = path.read_text()
+old = '\tprintf \'%s\\n\' "${state}${markers:+ (${markers})}"\n'
+new = (
+    '\tif [ -n "${markers}" ]; then\n'
+    '\t\tprintf \'%s\\n\' "${state} (${markers})"\n'
+    '\telse\n'
+    '\t\tprintf \'%s\\n\' "${state}"\n'
+    '\tfi\n'
+)
+count = text.count(old)
+if count != 1:
+    raise SystemExit(f"AdGuardHome.sh: expected one status marker expression, found {count}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)